langchain/retrievers/document_compressors/cohere_rerank.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Dict, Sequence
+
+from pydantic import root_validator
+
+from langchain.retrievers.document_compressors.base import BaseDocumentCompressor
+from langchain.schema import Document
+from langchain.utils import get_from_dict_or_env
+
+
+class CohereRerank(BaseDocumentCompressor):
+    client: Any
+    top_n: int = 3
```

Comment:
we usually call this k/top_k, dunno if we prefer being consistent internally or w.r.t. the client api

Reply:
gonna pref top_n - if we had a standard reranking class may rename, but until then will keep separate (if we want to change in future, with pydantic alias it is easy)